_data/legal.yml

```
@@ -3,27 +3,91 @@
   description: Going forward, all updates to the Gruntwork terms of service, privacy policy, open source policy, and other policies relating to data processing will be posted here on our our RSS feed
   category: privacy-policy
   link: https://gruntwork.io/
```

Please update this link to the RSS feed link.
```suggestion
  link: https://gruntwork.io/legal.rss
```